deb/build/debian/jenkins.default

```diff
@@ -7,9 +7,14 @@ NAME=@@ARTIFACTNAME@@
 JAVA=/usr/bin/java
 
 # arguments to pass to java
-JAVA_ARGS="-Djava.awt.headless=true"  # Allow graphs etc. to work even when an X server is present
+
+# Allow graphs etc. to work even when an X server is present
+JAVA_ARGS="-Djava.awt.headless=true"
+
 #JAVA_ARGS="-Xmx256m"
-#JAVA_ARGS="-Djava.net.preferIPv4Stack=true" # make jenkins listen on IPv4 address
+
+# make jenkins listen on IPv4 address
+#JAVA_ARGS="-Djava.net.preferIPv4Stack=true"
```

Comment:
This seems like it could confuse users, as uncommenting this (a common pattern in configuration files with defaults) will _replace_ the earlier value set.